ddl/ddl_tiflash_api.go

```diff
@@ -530,25 +515,24 @@ func (d *ddl) refreshTiFlashTicker(ctx sessionctx.Context, pollTiFlashContext *T
 				continue
 			}
 
-			progress, err := getTiFlashTableSyncProgress(pollTiFlashContext, tb.ID, tb.Count)
+			progress, err := infosync.GetTiFlashProgress(tb.ID, tb.Count, pollTiFlashContext.TiFlashStores)
```

Comment:
So seems that the current pattern is, all TiDBs will pull progress periodically and store the progress in the cache. When querying the PROGRESS table, info is retrieved from the cache, right?

Reply:
Only the ddl owner will call the `refreshTiFlashTicker ` and store the progress in the cache. If user query `information.tiflash_replica` on other TiDB nodes, other TiDB will send cop request to  ddl owner node to get progress.